pandas/tests/frame/test_constructors.py

```diff
@@ -262,12 +262,6 @@ def test_emptylike_constructor(self, emptylike, expected_index, expected_columns
         tm.assert_frame_equal(result, expected)
 
     def test_constructor_mixed(self, float_string_frame):
-        index, data = tm.getMixedTypeDict()
-
-        # TODO(wesm), incomplete test?
-        indexed_frame = DataFrame(data, index=index)  # noqa
-        unindexed_frame = DataFrame(data)  # noqa
```

Comment:
did you make a decision to remove this or was it automated?  id be OK with the former

Reply:
I made the decision to remove this since it didn't seem clear that this would be fixed